manifests/p/PGPTool/PGPTool/0.5.9.2/PGPTool.PGPTool.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: PGPTool.PGPTool
+PackageVersion: 0.5.9.2
+PackageLocale: en-US
+Publisher: PGPTool

```

```suggestion
Publisher: Sergey Karpushin
```
This should match control panel